build-jenkins-agent-ubuntu.pkr.hcl

```diff
@@ -30,6 +30,32 @@ build {
     ]
   }
 
+  # Retrieve agent.jar
+  provisioner "shell" {
+    only = ["docker.ubuntu"]
+    environment_vars = concat(local.provisioning_env_vars, [
+      "LANG=en_US.UTF-8",
+      "LANGUAGE=en_US:en",
+      "LC_ALL=en_US.UTF-8",
+      "AGENT_WORKDIR=/home/jenkins/agent"
+    ])
+    inline = [
+      "echo Retrieve agent.jar",
+      "curl --create-dirs -fsSLo /usr/share/jenkins/agent.jar https://repo.jenkins-ci.org/public/org/jenkins-ci/main/remoting/${var.remoting_version}/remoting-${var.remoting_version}.jar",
```

Comment:
```suggestion
      "curl --create-dirs --fail --silent --show-error --location --output /usr/share/jenkins/agent.jar https://repo.jenkins-ci.org/public/org/jenkins-ci/main/remoting/${var.remoting_version}/remoting-${var.remoting_version}.jar",
```

Reply:
What is the reason to put the agent.jar file in this location (and then having to create a symlink) ?

Reply:
I've reproduced it from the original docker-agent: https://github.com/jenkinsci/docker-agent/blob/master/17/alpine/Dockerfile#L40-L43

And the entrypoint comes from https://github.com/jenkinsci/docker-inbound-agent/blob/master/17/alpine/Dockerfile#L38

Reply:
The simlink doesn't work as intended anyway, I'll copy it directly in /usr/local/bin
Should I keep a copy in /usr/share too?

Reply:
>  Should I keep a copy in /usr/share too?

I don't remenber see any feature of Jenkins or documentation mentioning that it should. Let's put it in /usr/local/bin like you suggested, and see if it works like it